openqaoa/devices.py

```diff
@@ -385,7 +389,13 @@ def device_class_arg_mapper(device_class:DeviceBase,
                             execution_timeout: float = None,
                             client_configuration: QCSClientConfiguration = None,
                             endpoint_id: str = None,
-                            engagement_manager: EngagementManager = None) -> dict:
+                            engagement_manager: EngagementManager = None,
+                            device_name: str = None,
+                            aws_access_key_id: str = None, 
+                            aws_secret_access_key: str = None,
+                            aws_region: str = None, 
+                            s3_bucket_name: str = None,
+                            folder_name: str = 'oq_runs') -> dict:
```

Comment:
Depends on the usage of people! Let's ask the ones who have used AWS backends the most. @shahidee44, @shaohenc suggestions?

Reply:
Since this involves actual money, would it be safer to not have any defaults? To avoid any unintended surprises.

Reply:
I think just having a generic default name for the folder can be useful. Assuming people will run their scripts without knowing much about the features of braket, the least we can do is arrange the runs nicely into a folder on theirs3 bucket. (Which is what the default folder name helps them achieve easily)

Reply:
@Q-lds, thoughts?

Reply:
sure, I'll change the name to `openqaoa`. Another thing, I'll remove the default value from the `__init__` method of the class (so that we have the default at only one place). Then you can create the AWS device with the default value, if you do that using the `create_device` method, but not if you use the `DeviceAWS` directly